ghcup

```diff
@@ -297,9 +297,11 @@ USAGE:
     ${SCRIPT} compile [FLAGS] <VERSION> <BOOTSTRAP-GHC>
 
 FLAGS:
-    -h, --help       Prints help information
-    -f, --force      Overwrite already existing installation
-    -j, --jobs <n>   How many jobs for compilation
+    -h, --help                     Prints help information
+    -f, --force                    Overwrite already existing installation
+    -j, --jobs <n>                 How many jobs for compilation
+    -c, --build-config <filepath>  Use the given config file as build config
+                                   (mk/build.mk)
```

Comment:
Btw, at first, the "(mk/buiild.mk)" seemed a bit confusing to me, as I wrongly assumed that'd be the default `<filepath>`